netmaster/objApi/apiController.go

```diff
@@ -1175,6 +1175,84 @@ func (ac *APIController) PolicyCreate(policy *contivModel.Policy) error {
 	return nil
 }
 
+// PolicyGetOper inspects policy
+func (ac *APIController) PolicyGetOper(policy *contivModel.PolicyInspect) error {
+	log.Infof("Received PolicyInspect: %+v", policy)
+
+	// Get the state driver
+	stateDriver, err := utils.GetStateDriver()
+	if err != nil {
+		return err
+	}
+
+	// To hold total number of Endpoint count
+	var policyEPCount int
+
+	policyCfg := &mastercfg.EpgPolicy{}
+	policyCfg.StateDriver = stateDriver
+
+	// Policy is attached to EPG. So we need to fetch EPGs as well.
+	epgCfg := &mastercfg.EndpointGroupState{}
+	epgCfg.StateDriver = stateDriver
+
+	// Get all the Endpoints
+	readEp := &mastercfg.CfgEndpointState{}
+	readEp.StateDriver = stateDriver
+	epCfgs, epErr := readEp.ReadAll()
+
+	// Get all the EPGs on which this policy is applied
+	epgs := policy.Config.LinkSets.EndpointGroups
+
+	// Scan all the EPGs which are under this policy
+	for _, epg := range epgs {
+		log.Infof("EPG Object : %+v", epg)
+
+		// Reversing key from TenantName:EPGName to EPGName:TenantName
+		sList := strings.Split(epg.ObjKey, ":")
+		if sList == nil {
+			log.Errorf("EPG key %+v is not in valid format", epg.ObjKey)
+			return err
+		}
+		epgID := sList[1] + ":" + sList[0]
+
+		log.Infof("EPG ID : %s", epgID)
+
+		if err := epgCfg.Read(epgID); err != nil {
+			log.Errorf("Error fetching endpointGroup from mastercfg: %s", epgID)
+			return err
+		}
+
+		policyEPCount = policyEPCount + epgCfg.EpCount
+
+		if epErr == nil {
+			for _, epCfg := range epCfgs {
+				ep := epCfg.(*mastercfg.CfgEndpointState)
+				if ep.EndpointGroupKey == epgID {
```

Comment:
As i mentioned in the contivmodel review comments, policy oper state should probably container just a the list of endpoint groups using the policy.. and endpoint group should have the list of endpoints..
Having said that, i can see the value of displaying list of endpoints using the policy in addition to list of endpoint groups using the policy..